src/main/java/service/HorizontalServiceImpl.java

```diff
@@ -6,11 +6,11 @@
 
     private static final double dollarExchangeRate = 15;
```

Comment:
you can extract it to .prorerties

Reply:
dollarExchangeRate added to application.properties